index.js

```diff
@@ -0,0 +1,140 @@
+const tripsURL = 'https://trektravel.herokuapp.com/trips';
+//
+
+$( ".new-trip" ).hide();
+
+// Status Management
+//
+const reportStatus = (message) => {
+  $('#status-message').html(message).fadeOut(5000);;
```

Comment:
`fadeOut` doesn't reset the style on the element, so after the first time it's faded all the way out, the box never appears again!